services/api/src/rsuinfo.py

```diff
@@ -50,6 +62,9 @@ def options(self):
         # CORS support
         return ("", 204, self.options_headers)
 
-    def get(self):
+    @require_permission(
+        required_role=ORG_ROLE_LITERAL.USER,
+    )
+    def get(self, permission_result: PermissionResult):
         logging.debug("RsuInfo GET requested")
-        return (get_rsu_data(request.environ["organization"]), 200, self.headers)
+        return (get_rsu_data(permission_result), 200, self.headers)
```

Comment:
question: I'm seeing calls to `get_rsu_data` mostly without providing the positional arg of `qualified_orgs`. Should we be passing an empty list in for `qualified_orgs` everywhere, remove the `qualified_orgs` arg, or is there another approach that makes most sense?